From 9766f342eee11a3ed5e0ecd9ebd86d9ad415987f Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:05:42 -0400
Subject: [usb] ftdi_sio: add Cressi Leonardo PID

Message-id: <1410887463-20674-271-git-send-email-dzickus@redhat.com>
Patchwork-id: 95192
O-Subject: [RHEL7 PATCH 270/591] USB: ftdi_sio: add Cressi Leonardo PID
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 6dbd46c849e071e6afc1e0cad489b0175bca9318
Author: Joerg Dorchain <joerg@dorchain.net>
Date:   Fri Feb 21 20:29:33 2014 +0100

    USB: ftdi_sio: add Cressi Leonardo PID

    Hello,

    the following patch adds an entry for the PID of a Cressi Leonardo
    diving computer interface to kernel 3.13.0.
    It is detected as FT232RL.
    Works with subsurface.

    Signed-off-by: Joerg Dorchain <joerg@dorchain.net>
    Cc: stable <stable@vger.kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/ftdi_sio.c b/drivers/usb/serial/ftdi_sio.c
index ce0d7b0..41afa92 100644
--- a/drivers/usb/serial/ftdi_sio.c
+++ b/drivers/usb/serial/ftdi_sio.c
@@ -904,6 +904,8 @@ static const struct usb_device_id id_table_combined[] = {
  /* Crucible Devices */
  { USB_DEVICE(FTDI_VID, FTDI_CT_COMET_PID) },
  { USB_DEVICE(FTDI_VID, FTDI_Z3X_PID) },
+ /* Cressi Devices */
+ { USB_DEVICE(FTDI_VID, FTDI_CRESSI_PID) },
  { }     /* Terminating entry */
 };
 
diff --git a/drivers/usb/serial/ftdi_sio_ids.h b/drivers/usb/serial/ftdi_sio_ids.h
index a7019d1..b2d43b6 100644
--- a/drivers/usb/serial/ftdi_sio_ids.h
+++ b/drivers/usb/serial/ftdi_sio_ids.h
@@ -1313,3 +1313,9 @@
  * Manufacturer: Smart GSM Team
  */
 #define FTDI_Z3X_PID  0x0011
+
+/*
+ * Product: Cressi PC Interface
+ * Manufacturer: Cressi
+ */
+#define FTDI_CRESSI_PID  0x87d0
-- 
1.7.1